### ISSUE TITLE
Debian GNU/Linux grains broken in head of 2015.8 branch

### ISSUE THREAD

**Issue by terminalmage** [saltstack/salt#31410]
885e00b broke this. I checked and this breakage did not hit 2015.8.7.

I have a fix done already, also working on a unit test.

**Comment by cachedout** [comment]
Ah, crap. I was worried about that. My fault. Thanks for getting this.

**Comment by terminalmage** [comment]
No prob. It looks like if we detect the OS as Debian instead of Debian GNU/Linux, this  isn't affected.

**Comment by jtand** [comment]
@terminalmage We detect the OS as `Debian` 

```yaml
salt deb8_minion grains.item os
deb8_minion:
    ----------
    os:
        Debian
```

**Comment by terminalmage** [comment]
Yeah, so this is why it snuck past Jenkins. I'll work on a unit test to keep this from regressing

**Comment by terminalmage** [comment]
Fixed in #31418, unit test added.
